Python/ceval.c

```diff
@@ -942,7 +939,8 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     }
 
 resume_with_error:
-    SET_LOCALS_FROM_FRAME();
+    next_instr = frame->instr_ptr; \
```

Comment:
No backslash needed.
```suggestion
    next_instr = frame->instr_ptr;
```